app/src/main/java/com/codesoom/assignment/domain/Tasks.java

```diff
@@ -0,0 +1,76 @@
+package com.codesoom.assignment.domain;
+
+import java.util.*;
+
+/**
+ * {@code Tasks} is the manager of task
+ *
+ * @author etff
+ * @version 1.0.0 21/02/29
+ * @see com.codesoom.assignment.domain.Task
+ */
+public class Tasks {
+    private Map<Long, Task> tasks;
+    private Long taskId = 1L;
+
+    public Tasks() {
+        this.tasks = new LinkedHashMap<>();
+    }
+
+    /**
+     * Add task to tasks
+     *
+     * @param task item of task
+     */
+    public void addTask(Task task) {
+        Long generateId = generateId();
+        task.setId(generateId);
+        tasks.put(generateId, task);
+    }
+
+    /**
+     * Remove task
+     *
+     * @param id target id of task
+     */
+    public void remove(Long id) {
+        this.tasks.remove(id);
+    }
+
+    /**
+     * Find target task
+     *
+     * @param id target task id
+     * @return targeted task
+     */
+    public Optional<Task> findTask(Long id) {
+        return Optional.ofNullable(tasks.get(id));
+    }
+
+    /**
+     * Find all saved tasks
+     *
+     * @return all saved tasks
+     */
+    public List<Task> getTasks() {
+        return new ArrayList<>(tasks.values());
+    }
+
+    /**
+     * Find size of collection
+     *
+     * @return size of saved task
+     */
+    public int size() {
+        return tasks.size();
+    }
+
+    /**
+     * Generate auto increase task id
```

Comment:
`auto`나 `increase` 같은 개념은 이 메소드의 구현과 관련된 내용입니다. 주석이 코드 구현을 알고 있다면, 메소드의 구현이 변경되었을 때 주석도 수정되어야 할 것입니다. 객체지향 원칙을 메소드를 설명하는 주석에도 적용할 수 있다는 것을 깨달으시면 많은 것이 새롭게 보일 거에요.

```suggestion
     * 아이디를 생성해 리턴합니다.
```

Reply:
가끔 주석에 구현방법등을 써두고는 했는데, 고쳐나가야겠습니다!